plugins/modules/storage/nxos_zone_zoneset.py

```diff
@@ -916,18 +916,27 @@ def main():
                             + " is not activated, hence cannot deactivate"
                         )
                 elif actionflag == "activate":
-                    messages.append(
-                        "activating zoneset '"
-                        + zsetname
-                        + "' in vsan "
-                        + str(vsan)
-                    )
-                    actcmd.append(
-                        "zoneset activate name "
-                        + zsetname
-                        + " vsan "
-                        + str(vsan)
-                    )
+                    if commands_executed:
+                        messages.append(
```

Comment:
Looks like this module also returns a key `messages` in the result. This has not been documented. Could you please add this [here](https://github.com/ansible-collections/cisco.nxos/blob/70b5242bc27961248df9a702ae712109f8911c05/plugins/modules/storage/nxos_zone_zoneset.py#L187)?

Reply:
sure i also noticed that the mds modules were kept in plugins/modules/storage/ but i see copy of it in plugins/modules/ as well, should i delete those?

Reply:
@srbharadwaj the ones in plugins/modules/ are just soft links to the ones in plugins/modules/storage.

Reply:
They don't seem to be picked up by ansible from within the storage folder directly, so we had to add those.